BeginnerGuide.md

```diff
@@ -0,0 +1,124 @@
+    - Open Settings by pressing ***Ctrl+Alt+S***  
+      ![](https://user-images.githubusercontent.com/119479391/212522659-2035f1f0-15de-4ae0-a6bd-a976d420f9d1.png)
+
+    - Follow the path ***IDE settings>Language & Frameworks > Lombok*** and then click on ***Lombok.***  
+      ![](https://user-images.githubusercontent.com/119479391/212522661-8529fb75-f547-4753-b5b3-af1bcaa5bcf3.png)
+
+    - Make sure that the ***Lombok processing*** is enabled.  
+      ![](https://user-images.githubusercontent.com/119479391/212544060-584a266b-9f22-4a16-a9d6-1a9531c1d4e4.png)
+
+- ### **Step 5: Reload the project**
+    - Open the ***Maven*** Tab  
+      ![](https://user-images.githubusercontent.com/119479391/212522678-4d2f4953-9bc7-4fdb-ab40-7739e13e4ede.png)
+
+    - Press the ***Reload*** button as shown below and allow the project to Reload.   
+      ![](https://user-images.githubusercontent.com/119479391/212522686-dd263257-972e-4e41-a0d5-fac630f3510f.png)
+
+- ### **Step 6: Running the Project.**
+    - Open the ***WrongSecretsApplication*** by following the path ***main>java>org.owasp.wrongsecrets>WrongSecretApplication***.  
+      ![](https://user-images.githubusercontent.com/119479391/212522695-3bb435b2-e57f-44ee-a9d5-f583a53e15f1.png)
+    - Press ***Shift+F10*** to run the application, this will open up the ***Run/Debug Configurations Menu.***  
+      ![](https://user-images.githubusercontent.com/119479391/212522690-0e8e50eb-9c40-42ab-a7cb-f3cc6fed02ae.png)
+
+- ### **Step 7: Setting up Configurations.**
+    -  Select ***Edit configuration templates*** then select ***Application*** section.  
+       ![](https://user-images.githubusercontent.com/119479391/212778146-162ff060-8e37-448e-8e04-9cbb803ee9f5.png)
+    - There under the ***Application*** section click on the button shown below.  
+      ![](https://user-images.githubusercontent.com/119479391/212889483-55e98365-aab9-432e-9c5c-177c0484cb16.png)
+    - ***Select*** all the fields that are Selected in the below picture.  
+      ![](https://user-images.githubusercontent.com/119479391/212891550-0ded65d2-f78d-4a83-82bf-c4bd44a07f71.png)
+    - ***Fill*** all the fields as shown below.   
+      ![](https://user-images.githubusercontent.com/119479391/212888252-3d554893-3c11-484c-9f26-84c34b0301e7.png)
+    - Again press ***Shift+F10*** which runs the Application.  
+      ![](https://user-images.githubusercontent.com/119479391/212522690-0e8e50eb-9c40-42ab-a7cb-f3cc6fed02ae.png)
+
+    - ### **There you have it, ***WrongSecretsApplication*** running successfully.**
+      Here is a *preview* on how does it look after successfully running the Application.  
+      **Note:** Running the Application doesn't open any kind of ***GUI***, it only initializes the ***local webserver*** that you can open via a ***browser.***    
+      <img width="1445" alt="image (1)" src="https://user-images.githubusercontent.com/119479391/214242239-f8d3ed69-6d03-44de-8768-2bc1be8ef3ef.png">
+
+
```

Comment:
So now we have a running application, can you maybe add something on how to actually make some changes? E.g., something in the HTML, or adding a challenge?

Reply:
I was Suggest not to use HTML in the guide
And About how to add a Challenge i can link this part of ReadMe file in the Guide?
![challange](https://user-images.githubusercontent.com/119479391/214452782-27d5e4c1-08ad-461d-9872-b869ee332042.png)

Reply:
@bendehaan did you mean using HTML meant to Make things more Attractive in the Front-End part of the WebServer?

Reply:
The last one, yes. I was thinking about an example where you would be changing something in the WrongSecrets code (e.g. in the HTML of the WrongSecrets application).